GPCS4/SceModules/SceLibkernel/sce_libkernel.h

```diff
@@ -19,6 +19,7 @@
 #include "sce_kernel_eventflag.h"
 #include "sce_kernel_tls.h"
 
+#include <semaphore.h>
```

Comment:
This can not be compiled on Windows, we should find a semaphore library like pthread4w or implement it ourselves.

Reply:
yes, right, it's part of pthreads4w, i've changed the path